core/src/main/scala/Monitoring.scala

```diff
@@ -160,10 +183,13 @@ trait Monitoring {
 
           urlSignals.put(source, hook)
 
+          val removeFromMirroringUrls: Process[Task, Unit] =
```

Comment:
Just curious, is this just factored out for readability? Or some technical reason?